gravitee-apim-repository/gravitee-apim-repository-jdbc/src/test/java/io/gravitee/repository/jdbc/JdbcTestRepositoryInitializer.java

```diff
@@ -194,7 +194,7 @@ public void tearDown() {
         jt.execute(
             (Connection con) -> {
                 for (final String table : tablesToTruncate) {
-                    jt.execute("truncate table " + escapeReservedWord(prefix + table));
+                    jt.execute("truncate table " + escapeReservedWord(prefix + table) + " CASCADE");
```

Comment:
It doesn't work with all DBMS. For example, running tests with MariaDB :
`org.springframework.jdbc.BadSqlGrammarException: StatementCallback; bad SQL grammar [truncate table `test_gio_apis` CASCADE]; nested exception is java.sql.SQLSyntaxErrorException: (conn=10) You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'CASCADE' at line 1
Caused by: java.sql.SQLSyntaxErrorException: (conn=10) You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'CASCADE' at line 1
Caused by: java.sql.SQLException: 
You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'CASCADE' at line 1
Query is: truncate table `test_gio_apis` CASCADE
`